concepts/05 UI Components/Gantt/20 Filter Data/1 Filter Row.md

```diff
@@ -0,0 +1,407 @@
+The **Gantt** component allows users to filter columns by filter row values. To apply filter criteria, users can enter a value in a column's filter row cell or select the value in a specific editor. 
```

Comment:
`in a specific editor` Which specific editor?

Reply:
To be honest, `specific` doesn't mean depending on the data type. Perhaps 'corresponding'? Even so, if there is only one editor, then you would omit it - 
```suggestion
The **Gantt** component allows users to filter columns by filter row values. To apply filter criteria, users can enter a value in a column's filter row cell or select the value in the editor. 
```

Reply:
```suggestion
The **Gantt** component allows users to filter columns by filter row values. To apply filter criteria, users can enter a value in a column's filter row cell or select the value in this cell's editor.
```